lib/spack/docs/environments.rst

```diff
@@ -9,139 +9,279 @@
+   environment, but may concretize differently.
+
+If you want a more exact reproduction of the existing environment,
+you can use the ``spack.lock`` lockfile by entering:
 
 .. code-block:: console
 
    $ spack env create myenv spack.lock
 
-Either of these commands can also take a full path to the
-initialization file.
+.. note::
+
+   A Spack environment created from a ``spack.lock`` lockfile
+   is guaranteed to have the same concrete specs as the original
+   environment.
 
-A Spack Environment created from a ``spack.yaml`` manifest is
-guaranteed to have the same root specs as the original Environment,
-but may concretize differently. A Spack Environment created from a
-``spack.lock`` lockfile is guaranteed to have the same concrete specs
-as the original Environment. Either may obviously then differ as the
-user modifies it.
 
-^^^^^^^^^^^^^^^^^^^^^^^^^
+.. _environments-activating:
+
+--------------------------
 Activating an Environment
-^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------------
+
+Once the environment has been created, you will need to activate it
+before you can use it.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Activating a Managed Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To activate an environment, use the following command:
+To activate the managed environment, use the following command:
 
 .. code-block:: console
 
    $ spack env activate myenv
 
-By default, the ``spack env activate`` will load the view associated
-with the Environment into the user environment. The ``-v,
---with-view`` argument ensures this behavior, and the ``-V,
---without-view`` argument activates the environment without changing
-the user environment variables.
+or the shortcut alias
+
+.. code-block:: console
+
+   $ spacktivate myenv
+
+By default, ``spack env activate`` will load the view associated
+with the environment into the user environment. 
+
+Options affecting the user environment are:
 
-The ``-p`` option to the ``spack env activate`` command modifies the
-user's prompt to begin with the environment name in brackets.
+* ``-v, --with-view``:  ensures the environment view is loaded
+* ``-V, --without-view``: activates the environment without changing
+  the user environment variables
+
+There is another handy option for those who like to modify their
+prompts. The ``-p`` option changes the user's prompt to begin with
+the environment name in brackets.
 
 .. code-block:: console
 
    $ spack env activate -p myenv
    [myenv] $ ...
 
-To deactivate an environment, use the command:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Activating an Independent Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Recall that independent environments are not managed by Spack or
+accessed by name; rather, they are identified by their directory.
+So, activating an independent environment simply requires the
+``-d </path/to/myenv/directory>`` arguments.  For example:
+
+.. code-block:: console
+
+   $ spack env activate -d /path/to/myenv/directory
```

Comment:
You don't actually need the `-d` argument here, which makes the distinction between this section and the previous seem unnecessary. You can list a path or name directly.